app/dao/organisation_dao.py

```diff
@@ -158,3 +158,22 @@ def dao_add_user_to_organisation(organisation_id, user_id):
 @autocommit
 def dao_remove_user_from_organisation(organisation, user):
     organisation.users.remove(user)
+
+
+@autocommit
+def dao_add_email_branding_to_organisation_pool(organisation_id, email_branding_id):
+    organisation = dao_get_organisation_by_id(organisation_id)
+    email_branding = EmailBranding.query.filter_by(id=email_branding_id).one()
+    email_branding.organisation.append(organisation)
```

Comment:
Optional: You could consider changing this line to:

`organisation.email_branding_pool.append(email_branding)`

That way you are doing what the function name says, you are adding email branding to the organisations branding pool (rather than adding the organisation to the list of orgs using that email branding)

Caveat, I haven't tested the above to check it works

Reply:
Cool, that sounds sensible, we will see if it works alright 👍🏼

Reply:
Gozie made it work, so yay done now ☺️